scripts/build/Jenkins/Jenkinsfile

```diff
@@ -586,24 +609,26 @@ def CreateSingleNode(Map pipelineConfig, def platform, def build_job, Map envVar
                         if (build_job_name.toLowerCase().contains('asset') && env.IS_UPLOAD_AP_LOGS?.toBoolean()) {
                             CreateUploadAPLogsStage(platform.key, build_job_name, envVars['WORKSPACE'], platform.value.build_types[build_job_name].PARAMETERS).call()
                         }
+                        // Upload test artifacts only on builds that failed and ran test suites
+                        if (env.IS_UPLOAD_TEST_ARTIFACTS && params.containsKey('CMAKE_TARGET') && params.CMAKE_TARGET.contains("TEST_SUITE")) {
```

Comment:
This should be env.IS_UPLOAD_TEST_ARTIFACTS?toBoolean()

Reply:
fixed